db/migrate/20180220180926_add_default_value_to_comment_status.rb

```diff
@@ -0,0 +1,6 @@
+class AddDefaultValueToCommentStatus < ActiveRecord::Migration
+  def change
+    change_column :comments, :status, :integer, default: 1
```

Comment:
Are you just changing the default here then? Can you add a comment that only the default is changing?

Otherwise this looks good and it'll need an update to the schema.rb.example to pass tests. Good work!

Reply:
>Are you just changing the default here then? Can you add a comment that only the default is changing?

Both default value and existing comment's status are updating with this as we have discussed